
test('Our first test', ()=>{
    throw new Error('something went wrong!');
});